.github/workflows/checklistMarkdownToPDF.yml

```diff
@@ -0,0 +1,65 @@
+name: Converting outbound checklists from .md to .pdf
+on:
+    pull_request:
+        types: [opened, synchronize]
+    # Paths can be used to only trigger actions when you have edited checklist files
+        paths:
+        - 'tier*/checklist.md'
+
+jobs:
+  get-changed-directories:
+    name: Get changed directories
+    runs-on: ubuntu-latest
+    outputs:
+      tiers: ${{ steps.list-dirs.outputs.tiers }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+      - name: Get changed directories
+        id: get-dirs
+        uses: tj-actions/changed-files@v45
+        with:
+          dir_names: "true"
+      - name: List all changed tier directories
+        id: list-dirs
+        env:
+          ALL_CHANGED_FILES: ${{ steps.get-dirs.outputs.all_changed_files }}
+        run: |
+          # Obtain changed tier directories and format into array
+          DIRS=$(echo "$ALL_CHANGED_FILES" | grep -oE 'tier[^ ]*' | sed 's/^/"/; s/$/"/' | paste -sd, -)
+
+          # Output the array
+          TIER_DIRS="[$DIRS]"
+          echo "$TIER_DIRS"
+
+          echo "tiers=$TIER_DIRS" >> "$GITHUB_OUTPUT" 
+
+  convert-to-pdf:
+    name: Build PDF
+    runs-on: ubuntu-latest
+    needs: get-changed-directories
+    permissions:
+        contents: write
+    strategy:
```

Comment:
We might run into an issue here when trying to push simultaneously to this repo in a matrix. If the various parts of the matrix push and become out of date with remote it could cause problems and merge conflicts that cause the job to fail.

In the git-auto-commit-action workflow, there are limitations that say that it won't handle merges or pulling if the remote is out of date: https://github.com/stefanzweifel/git-auto-commit-action?tab=readme-ov-file#limitations--gotchas

Reply:
Good flag, done!